temporalcli/commandsmd/commands.md

```diff
@@ -815,9 +815,8 @@ temporal workflow execute
 
 #### Options
 
-* `--event-details` (bool) - If set when using text output, this will print the event details instead of just the event
-  during workflow progress. If set when using JSON output, this will include the entire "history" JSON key of the
-  started run (does not follow runs).
+* `--event-details` (bool) - If set when using text output, include event details JSON in printed output.
+  If set when using JSON output, this will include the entire "history" JSON key of the started run.
```

Comment:
This was drive-by editing. It seemed to me that the presence of the flag was not related to whether it follows runs.